network/p2p/dht.go

```diff
@@ -0,0 +1,67 @@
+package p2p
+
+import (
+	"context"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	discovery "github.com/libp2p/go-libp2p-discovery"
+	dht "github.com/libp2p/go-libp2p-kad-dht"
+)
+
+const routingTableRefresh = time.Second // * 100
+const peerDiscoveryTimeout = time.Second * 2
```

Comment:
How did we decide on these values? Would it make sense to do some test to verify that these are actually good values to use?

Reply:
yeah - thats a good idea. I haven't tested these tbh, I just take them from [here](https://github.com/onflow/flow-go/pull/1052)

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/blob/3cefab2d2376cdb98d2011067265b7783500278e/internal/config/config.go#L113-L114

They are quite different from the defaults defined here, so I'm a bit concerned

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/blob/3cefab2d2376cdb98d2011067265b7783500278e/internal/config/config.go#L113-L114

They are quite different from the defaults defined here, so I'm a bit concerned

Reply:
actually, I would rather run with the defaults and adjust it later if we find an issue. I will remove these settings if y'all ok with it

Reply:
Yeah, let's just try the defaults for now. Later on we can do more testing or change it if we notice any issues.